src/porepy/compositional/materials.py

```diff
@@ -446,6 +423,36 @@ class SolidConstants(Constants):
     well_radius: number = 0.1
 
 
+@dataclass(kw_only=True, eq=False)
+class NumericalConstants(Constants):
+    """Data class containing numerical method parameters,
+    including characteristic sizes.
+    """
+
+    SI_units: ClassVar[dict[str, str]] = dict(
+        {
+            "characteristic_displacement": "m",
+            "characteristic_contact_traction": "Pa",
+            "open_state_tolerance": "-",
+            "contact_mechanics_scaling": "-",
+        }
+    )
+
+    characteristic_contact_traction: number = 1.0
+
+    characteristic_displacement: number = 1.0
+
+    contact_mechanics_scaling: number = 1e-1
+    """Safety scaling factor, making fractures softer than the matrix"""
+
+    open_state_tolerance: number = 1e-5
+    """Tolerance parameter for the tangential characteristic contact mechanics.
+
+    FIXME: Revisit the tolerance.
```

Comment:
@IvarStefansson With characteristic scaling factors for tractions, is this FIXME now obsolete?

Reply:
Almost. I think we should consider tightening it, given that it now compares values which we have scaled towards unity.

Reply:
I like the idea. Any preferred value? 1e-8?